src/_data/sidenav.yml

```diff
@@ -121,6 +121,11 @@
     - title: Platform integration
       permalink: /docs/development/platform-integration
       children:
+        - title: Adding Flutter to an Android app
```

Comment:
Don't have to do anything now but I think instead of grouping by platform, we might end up needing to group by function since some of the future guides like 'should I prewarm an engine or not', 'how do I let engines talk to each other' etc will be cross-platform docs.